### ISSUE TITLE
Mono 'The 'ExeConfigFilename' argument cannot be null.'

### ISSUE THREAD

**Issue by Martin-Molinero** [pythonnet/clr-loader#8]
Hi all,
We've recently updated https://github.com/QuantConnect/pythonnet/pull/47 to be aligned with https://github.com/pythonnet/pythonnet and started getting the following error
```
The 'ExeConfigFilename' argument cannot be null.
  at System.Configuration.ExeConfigurationHost.CheckFileMap (System.Configuration.ConfigurationUserLevel level, System.Configuration.ExeConfigurationFileMap map) [0x0001b] in <65bca3ec97b145069173491f1a974cbf>:0 
  at System.Configuration.ExeConfigurationHost.InitForConfiguration (System.String& locationSubPath, System.String& configPath, System.String& locationConfigPath, System.Configuration.Internal.IInternalConfigRoot root, System.Object[] hostInitConfigurationParams) [0x00030] in <65bca3ec97b145069173491f1a974cbf>:0 
  at System.Configuration.InternalConfigurationSystem.InitForConfiguration (System.String& locationConfigPath, System.String& parentConfigPath, System.String& parentLocationConfigPath) [0x00000] in <65bca3ec97b145069173491f1a974cbf>:0 
  at System.Configuration.Configuration..ctor (System.Configuration.InternalConfigurationSystem system, System.String locationSubPath) [0x0001f] in <65bca3ec97b145069173491f1a974cbf>:0 
  at System.Configuration.InternalConfigurationFactory.Create (System.Type typeConfigHost, System.Object[] hostInitConfigurationParams) [0x0000d] in <65bca3ec97b145069173491f1a974cbf>:0 
  at System.Configuration.ConfigurationManager.OpenExeConfigurationInternal (System.Configuration.ConfigurationUserLevel userLevel, System.Reflection.Assembly calling_assembly, System.String exePath) [0x000ef] in <65bca3ec97b145069173491f1a974cbf>:0 
  at System.Configuration.ClientConfigurationSystem.get_Configuration () [0x0000e] in <65bca3ec97b145069173491f1a974cbf>:0 
```

Steps to reproduce:
```
from pythonnet import load
from clr import AddReference
AddReference("RestSharp")
from RestSharp import *
client = RestClient("https://www.google.com/")
restsharpResponse = client.Execute(RestRequest())
print(restsharpResponse.ErrorException.InnerException)
```

I suspect it's related to this new `clr-loader`, seen some old related issues https://github.com/pythonnet/pythonnet/commit/37a4ea597631266c44593db2a365f9df58c8c06e & https://stackoverflow.com/questions/35872504/pythonnet-exception-the-execonfigfilename-argument-cannot-be-null .
- Seems `mono_domain_set_config` isn't being called anymore? should it be called automatically **internally** (as before I believe) or we are missing some call for it in? Or is it maybe we are screwing it up in the way we consume it/missing some step

Thanks!

**Comment by Martin-Molinero** [comment]
From my understanding `mono_domain_set_config` has to be called, maybe internally setting it 🤔 , see
> You need to add a call to mono_domain_set_config. It is not automatic. Though it doesn't really matter what you supply as the config file name it doesn't need to exist.

https://xamarin.github.io/bugzilla-archives/51/51261/bug.html

**Comment by C-SELLERS** [comment]
Request to close this because of #9 

Also if possible can we do a version bump and publish the latest?

**Comment by filmor** [comment]
Fixed in 0.1.6, thank you very much :)
